packages/eas-cli/src/commandUtils/flags.ts

```diff
@@ -0,0 +1,19 @@
+import { Flags } from '@oclif/core';
+
+export const EasNonInteractiveFlags = {
+  nonInteractive: {
```

Comment:
The naming here is not really clear, it suggests that it's about the noninteractive mode, but only one option has that param.

Maybe st like this?
```
export const EasJsonOutputFlags = {
  withNonInteractive: {
  ...
  jsonOnly: {
```

or maybe drop that wrapper object and just have separate values
```
export const EasNonInteractiveFlag ...
export const EasJsonOnlyFlag...
export const EasNonInteractiveAndJsonFlags...
```

Reply:
sgtm